pkg/audit/manager.go

```diff
@@ -121,11 +121,23 @@ func (am *Manager) audit(ctx context.Context) error {
 	// get updatedLists
 	updateLists := make(map[string][]auditResult)
 	totalViolationsPerConstraint := make(map[string]int64)
+	totalViolationsPerEnforcementAction := make(map[util.EnforcementAction]int64)
```

Comment:
nit: It looks like `getUpdateListsFromAuditResponses(resp)` already returns appropriate zero values, so we could drop `len(resp.Results()) > 0`.

We could think about simplifying the signature by updating the metrics in-function, but that may be surprising if the function gets called elsewhere.

Reply:
> we could drop if len(resp.Results()) > 0

done